src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -678,6 +705,18 @@ int sd_dhcp_lease_save(sd_dhcp_lease *lease, const char *lease_file) {
         if (r >= 0)
                 serialize_dhcp_routes(f, "ROUTES", routes, r);
 
+        r = sd_dhcp_lease_get_vendor_specific(lease, &data, &data_len);
```

Comment:
Why the single-use function here? Do you intend to use that from anywhere else? If not, its code should just be folded into here,